app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return 'All friends should be vaccinated'
+        except NotWearingMaskError:
+            total += 1
```

Comment:
This can be simplified to
```python
total = 0
for friend in friends:
         try:
             cafe.visit_cafe(friend)
         except (NotVaccinatedError, OutdatedVaccineError):
             return 'All friends should be vaccinated'
         except NotWearingMaskError:
             total += 1
```